src/frontend/packages/lib_components/src/utils/errors/report.tsx

```diff
@@ -4,7 +4,9 @@ import { useSentry } from '@lib-components/data/stores/useSentry';
 
 export const report = (error: unknown) => {
   if (useSentry.getState().isSentryReady) {
-    Sentry.captureException(error);
+    if (!(error instanceof TypeError && error.message === 'Load failed')) {
+      Sentry.captureException(error);
```

Comment:
This should not be done here but in the `sentry.init`. Sentry is initialized in the hook `useSentry`. You should use the filtering mechanism available and describe here: https://docs.sentry.io/platforms/javascript/configuration/filtering/ with the `beforeSend` callback probably.